### PR TITLE
feat: #4773 eject to react code

### PR DESCRIPTION
**Pull request by joshbalfour**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4773

### CODE REVIEW

File: packages/app-builder-backend/src/eject/templates/app.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { js } from './js'
+import { lint } from './format'
+
+export const generateApp = () => {
```
Comment (willmcvay) [inline packages/app-builder-backend/src/eject/templates/app.ts:4]: Feels a bit fragile / hard to maintain. I think in the future, we should do our own CRA template similar to the foundations one that has all this stuff pre-built

Comment (willmcvay) [reply packages/app-builder-backend/src/eject/templates/app.ts:4]: Not for today though!

File: packages/app-builder-backend/src/eject/templates/packageJson.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        graphql: '^15.5.1',
+        'react-router': '^5.2.0',
+        'react-router-dom': '^5.2.0',
+        typescript: '^4.4.2',
+
+        'omit-deep': '^0.3.0',
+        'query-string': '^7.0.1',
+        'clone-deep': '^4.0.1',
+
+        '@reapit/connect-session': '^3.0.0', // TODO: should update dynamically to latest version
```
Comment (willmcvay) [inline packages/app-builder-backend/src/eject/templates/packageJson.ts:22]: Can you just use @latest ? we flag our NPM releases

**Review by willmcvay — APPROVED** [review]
There's a lot in there but looks good to me!

File: packages/app-builder-backend/src/eject/templates/packageJson.ts (hunk 2 above)
Comment (joshbalfour) [reply packages/app-builder-backend/src/eject/templates/packageJson.ts:22]: unfortunately when you do that it comes back with this :/ 
image

**Pull request merged**